tests/unit/test_pandas/test_dataframe_empty.py

```diff
@@ -13,11 +13,13 @@
     ],
 )
 def test_empty(test_data):
-    profile = ProfileReport(test_data, progress_bar=False)
-    description = profile.get_description()
+    with pytest.raises(ValueError):
+        profile = ProfileReport(test_data, progress_bar=False)
+    
+        description = profile.get_description()
 
-    assert len(description["correlations"]) == 0
-    assert len(description["missing"]) == 0
+        assert len(description["correlations"]) == 0
+        assert len(description["missing"]) == 0
 
-    html = profile.to_html()
-    assert "Dataset is empty" in html
+        html = profile.to_html()
+        assert "Dataset is empty" in html
```

Comment:
this part of the code is not necessary anymore you can remove it, and that is it I believe.

Reply:
Alright :)